SConstruct

```diff
@@ -145,7 +145,7 @@ test = env.Program(
 	 # Add Catch header & additional test includes to the existing search paths
 	CPPPATH=(env.get('CPPPATH', []) + [pathjoin('tests', 'include')]),
 	# Do not link against the actual implementations of SDL, OpenGL, etc.
-	LIBS=sys_libs,
+	LIBS=sys_libs + game_libs,
```

Comment:
This looks very similar to what was discussed in [a DistanceMap unit test](https://github.com/endless-sky/endless-sky/pull/6142/files#r677357769). If the change is the same, then I expect the discussion there to apply here as well.

Reply:
You can't both use libraries in code and not link them. If you don't want to link to those libs alter the implementation to not use them or provide shims.

Reply:
For unit-tests I agree with tehhowch; you want unit-tests to run without external dependencies.
But there are many test-levels above unit-testing where it would be okay to link in external libraries. Just don't try to modify the unit-testing system with adding external libraries, but try to setup a "higher-level" testing system that can properly deal with external libraries. (or as you said, provide shims/mocks instead for the unit-tests)

Reply:
Please ping me once you have integration tests set up, I can look into the image tests then. Feel free to close this pr in such case, I guess